contentcuration/contentcuration/static/js/edit_channel/router.js

```diff
@@ -11,7 +11,6 @@ ChannelEditRouter  = Backbone.Router.extend({
     initialize: function(options) {
         _.bindAll(this, "navigate_channel_home", "preview_page", "edit_page", "clipboard_page");
 		this.nodeCollection = new Models.ContentNodeCollection();
-		this.nodeCollection.fetch();
```

Comment:
@jayoshih Please let me know if this was required for some reason. It loads *all* content nodes (across all channels), which is slow on the live server, and I couldn't figure out where it's used, so I removed it here. Everything still seems to work for me.